_data/tools.yml

```diff
@@ -1638,3 +1638,13 @@
   v3: true
   link: https://github.com/hotmeteor/spectator
   github: https://github.com/hotmeteor/spectator
+
+- name: APIFuzzer
+  category: data-validators
+  language: Python
+  description: Fuzz test your application using your OpenAPI or Swagger API definition without coding. Integrate into CI/CD, get Junit XML test result and
```

Comment:
can you remove `swagger` here? we are trying hard to remove that word from association here :)